### PR TITLE
Minor documentation fixes

### PR DESCRIPTION
**Pull request by cromedome**
I am migrating some apps at `$work` from D1 to D2, and caught a few things in the documentation. 

My intent is to help reconcile these against the manual rewrite branch in the not-too-distant future.

### CODE REVIEW

File: lib/Dancer2/Tutorial.pod [hunk 1]
```diff
@@ -629,13 +629,20 @@ match routes with regular expressions, or you can match pieces of a route
...
+shapes and sizes, and you can easily build a solution tailored to your 
+particular needs. 
+
+There is certainly a lot left to cover. We suggest you take a stroll through 
+the L<manual|Dancer2::Manual>for more detailed information about the 
+topics covered in this tutorial, and for a more thorough explanation of all 
+things Dancer2-related. The L<cookbook|Dancer2::Cookbook> will show you how 
+to handle some handy tips and tricks when working with Dancer2. Additionally, 
+there are a lot of great Dancer2 plugins which extend and enhance the 
+capabilities of the framework.
```
Comment (SysPete) [inline lib/Dancer2/Tutorial.pod]: maybe link to a metacpan search for plugins here?

**Comment by SysPete** [comment]
:+1: from me (though perhaps adding the link I mentioned in comment would be good)

File: lib/Dancer2/Tutorial.pod (hunk 1 above)
Comment (cromedome) [reply lib/Dancer2/Tutorial.pod]: Oooh, good idea, thanks.

Comment (cromedome) [reply lib/Dancer2/Tutorial.pod]: Done!

**Pull request closed without merging**